src/builtins/loaders/node/js-loader.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function loadJSBundle(bundle) {
+  return new Promise(function(resolve) {
+    // require in the next event loop tick to let the main module load
+    setTimeout(function() {
+      require(__dirname + bundle)
+      resolve()
```

Comment:
Should resolve to the return of require. `resolve(require(__dirname + bundle))`

Reply:
Are you sure? The original `js-loader.js` doesn't resolve anything : 
https://github.com/parcel-bundler/parcel/blob/0dc062f77e36a19297dac542be33cded04580e89/src/builtins/loaders/js-loader.js#L13-L16

If `require` is called here it's just to register the `__parcel_require` so next `require` calls can require numeric IDs here : 

https://github.com/parcel-bundler/parcel/blob/0dc062f77e36a19297dac542be33cded04580e89/src/builtins/bundle-loader.js#L27-L30

I'm not very familiar with that part though so I may have missed something 😅